src/status_im/extensions/core.cljs

```diff
@@ -35,6 +35,11 @@
  (fn [_ [_ _ m]]
    {::identity-event m}))
 
+(re-frame/reg-event-fx
+ :chat/send-message
+ (fn [cofx [_ message-text {:keys [to value] :as arguments}]]
```

Comment:
message-text not used? same for arguments

Reply:
oh yeah I was using those for debug logs. I can remove them.